docs/ert/reference/configuration/keywords.rst

```diff
@@ -155,6 +156,37 @@ Example:
     See the :ref:`DATA_KW <data_kw>` keyword which can be used to utilize more template
     functionality in the Eclipse/flow datafile.
 
+
+DESIGN_MATRIX
+-------------
+.. _design_matrix:
+
+DESIGN_MATRIX is meant to be used to read, validate and internalize into ert storage a design matrix parameters given in XLSX-format.
+:code:`DESING_MATRIX` supports 1 positional argument and up to two named arguments:
+
+1. Positional argument: XLSX file containing design and default sheet with parameters
+
+Optional named arguments:
+
+1. The name of the design sheet (defaults to DesignSheet)
+
+2. The name of the default sheet (defaults to DefaultSheet)
+
+*Example:*
+
+::
+
+        -- DESIGN_MATRIX poly_design.xlsx DESIGN_SHEET:DesignSheet DEFAULT_SHEET:DefaultSheet
```

Comment:
Suggest starting with an example with just a positional argument, and then have this example afterwards.